app/services/users/delete_account_service.rb

```diff
@@ -73,7 +73,9 @@ def create_audit_record
         metadata: {
           email: @user.email,
           account_deletion_notification_sent_at:
-            @user.account_deletion_notification_sent_at&.iso8601
+            @user.account_deletion_notification_sent_at&.iso8601,
+          cohort_ids: @user.founders.map(&:cohort_id).uniq,
```

Comment:
This can be improved by keeping the full op in the DB call.

```suggestion
          cohort_ids: @user.founders.distinct(:cohort_id).pluck(:cohort_id),
```

Reply:
Is distinct required here?